### PR TITLE
Update dialogue-hauntedhouse.js

### PR DESCRIPTION
**Pull request by A-Nightsparrow**

### CODE REVIEW

**Review by reaganhenke — APPROVED** [review]
@LaChapeliere I'm not sure what your comment refers to, but these changes look ok to me. Adding an approval so you can merge.

File: dialogue-hauntedhouse.js [hunk 1]
```diff
@@ -208,7 +208,7 @@ const hauntedHouseDialogue2 = [
         nextText: 14,
       },
       {
-        text: "(Lie) She wants you to find loving owners for the house, that will also adopt her cats.",
+        text: "(Lie) She wants you to find a loving owner for the house, someone who will also adopt her cats.",
```
Comment (reaganhenke) [inline dialogue-hauntedhouse.js]:
```suggestion
        text: "(Lie) She wants you to find a loving owner for the house, who will also adopt her cats.",
```
@LaChapeliere is right, this ran over a bit. Adjusting to fit. 
Screen Shot 2022-02-05 at 10 53 47 AM

Comment (LaChapeliere) [reply dialogue-hauntedhouse.js]: I don't know how to add a suggestion to your suggestion, but there just need to be a comma between "house" and "who". Apart from that, this text works for me :)

Comment (reaganhenke) [reply dialogue-hauntedhouse.js]: @LaChapeliere Updated, if that looks good to you I'll commit it in and merge this PR

**Pull request merged**